src/common/utils/UnhandledExceptionHandler_x64.h

```diff
@@ -64,7 +61,7 @@ static inline const char* exceptionDescription(const DWORD& code)
 }
 
 /* Returns the index of the last backslash in the file path */
-inline int GetFilenameStart(CHAR* path)
+inline int GetFilenameStart(wchar_t* path)
```

Comment:
This function should check for wide versions of characters, i.e. L'\\' and L'\0'

Reply:
You’re right, thank you!